src/seedData.ts

```diff
@@ -0,0 +1,114 @@
+import 'reflect-metadata'
+import { createConnection, Connection } from 'typeorm'
+
+import { faker } from '@faker-js/faker'
+import { Author } from '@/entity/author.entity'
+import { Genre } from '@/entity/genre.entity'
+import { Book } from '@/entity/book.entity'
+import { BookInstance } from '@/entity/book-instance.entity'
+
+async function seed() {
+  try {
+    const connection: Connection = await createConnection()
+    console.log('Connected to the database.')
+
+    const authors = await seedAuthors(connection)
+    const genres = await seedGenres(connection)
+    const books = await seedBooks(connection, authors, genres)
+    await seedBookInstances(connection, books)
+
+    await connection.close()
+    console.log('Seed data successfully!')
+  } catch (error) {
+    console.error('Error seeding data:', error)
+  }
+}
+
+async function seedAuthors(connection: Connection): Promise<Author[]> {
+  const authorRepository = connection.getRepository(Author)
+  const authors: Author[] = []
+
+  for (let i = 0; i < 5; i++) {
+    const author = new Author()
+    author.firstName = faker.person.firstName()
+    author.familyName = faker.person.lastName()
+    author.dateOfBirth = faker.date.past()
+    author.dateOfDeath = faker.date.future()
+    author.url = faker.image.avatarLegacy()
+    await authorRepository.save(author)
+    authors.push(author)
+  }
```

Comment:
```suggestion
    const author = new Author({
        firstName: faker.person.firstName(),
        familyName: faker.person.lastName(),
        ...
    })
    await authorRepository.save(author)
    authors.push(author)
  }
```
Tương tự cho các entity khác